sentry_sdk/scope.py

```diff
@@ -849,6 +849,10 @@ def set_tag(self, key, value):
         """
         self._tags[key] = value
 
+    def set_tags(self, tags):
+        # type: (Mapping[str, object]) -> None
+        self._tags.update(tags)
```

Comment:
Wondering about the `set_tags({})` case. Since we're using `dict.update()` in the background, `set_tags({})` would do nothing.

I can imagine folks being confused by this since if someone does `set_tags({})` explicitly, they probably expect the tags to be cleared. At the same time, I'm not sure if that's a valid usecase in the first place since they can just push a new scope, apply the tags they want, and then pop the scope again, clearing the tags. So I guess this is fine.

Reply:
While I understand your point, per the [unified API](https://develop.sentry.dev/sdk/unified-api/), the `set_tags` function should be a convenience function for replacing multiple `set_tag` calls. Since `set_tag` cannot remove/clear tags, I think the correct behavior is for `set_tags({})` to be a no-op. This also maintains consistency with calling `set_tags` with a non-empty dict, since we only update the values corresponding to keys in the dictionary passed.

I will add a docstring to this function to clarify the behavior.